elastalert/alerters/teams.py

```diff
@@ -36,7 +36,8 @@ def populate_facts(self, matches):
         alert_facts = []
         for arg in self.ms_teams_alert_facts:
             arg = copy.copy(arg)
-            arg['value'] = lookup_es_key(matches[0], arg['value'])
+            matched_value = lookup_es_key(matches[0], arg['value'])
+            arg['value'] = matched_value if matched_value else arg['value']
```

Comment:
Thanks for contributing this feature!

Suppose a config such as the following is used:
```
- name: Speed
  value: vehicle.speed
```
Now suppose that the alerting document contains an entry such as:
```
vehicle.speed:0
```

I would expect the MS Teams fact to be:
```
  Speed: 0
```

Can you confirm this behaves as expected for values such as 0, False, "", etc? A Unit test proving these scenarios should be included.

Reply:
It didn't work as you suspected. I have fixed it by replacing None checker to 'is not None' and added some more test cases.